routes/web.php

```diff
@@ -2,11 +2,12 @@
 
 use Illuminate\Support\Facades\Route;
 use App\Http\Controllers\ExampleController;
+use App\Http\Controllers\StoreController;
 
 // Route::get('login', [ExampleController::class, 'login']);
 // Route::get('cv', [ExampleController::class, 'cv']);
 Route::get('task3', [ExampleController::class, 'task3']);
-Route::post('user_data', [ExampleController::class, 'store'])->name('user_data');
+Route::post('user_data', [StoreController::class, 'store'])->name('user_data');
```

Comment:
you could have done it by much cleaner way..

Think of it

the function that show the form of contact us and the function that show the data inside the form,, is there is any correlation between them but them in the same controller

and us both of them handles some sort of work with contact us